src/Messages/Interactions/TSMessage.m

```diff
@@ -102,6 +102,7 @@ - (instancetype)initWithTimestamp:(uint64_t)timestamp
     _expiresInSeconds = expiresInSeconds;
     _expireStartedAt = expireStartedAt;
     [self updateExpiresAt];
+    _receivedAtDate = [NSDate date];
```

Comment:
> I'm not that in love with createdAt you could pick something else, I'm just easily confused by the nuances of a field called recordedAtDate on an object which already has a field called recordedAt which is a date.

I'm hoping that this isn't an issue any more, now that I've removed the `recordedAt` property?